outputs.tf

```diff
@@ -71,6 +49,41 @@ output "security_group_policies" {
   })
 }
 
+output "kubernetes_secrets" {
```

Comment:
I will maybe just drop a small comment here explaining that the purpose of this. That it's a helper for the guide and totally optional.
We are explaining that some TF vars are useless in vars descriptions if we don't use this output. I think it's also worth explaining what is this output about.

Reply:
Yeah, makes sense.